### PR TITLE
[#60] Enable temporal coverage validation

### PR DESCRIPTION
**Pull request by Maurier**
Enable crossover validation of temporal coverage start and end date.

This wasn't working properly in Vue 2 because of errors in transpiled dependencies, but it is no longer a problem with the new build tools in Vue 3.

image

### CODE REVIEW

File: api/models/schema.py [hunk 1]
```diff
@@ -12,549 +12,580 @@
...
 
 class TemporalCoverage(SchemaBaseModel):
-    startDate: datetime = Field(
-        title="Start date",
-        description="A date/time object containing the instant corresponding to the commencement of the time "
-                    "interval (ISO8601 formatted date - YYYY-MM-DDTHH:MM).",
-        # TODO: these are failing due to a problem with transpiled dependencies inside cznet-vue-core
-        # formatMaximum={"$data": "1/endDate"},
-        # errorMessage= { "formatMaximum": "must be lesser than or equal to End date" }
+  startDate: datetime = Field(
```
Comment (Maurier) [inline api/models/schema.py]: @pkdash - All other changes in this file are from linter, these are the only real changes.

Comment (pkdash) [reply api/models/schema.py]: What linter are you using? I think the standard indention is 4 spaces which it had originally. Now it seems using 2 spaces for indentation.

Comment (Maurier) [reply api/models/schema.py]: I'm using autopep8

`
"[python]": {
      "editor.formatOnType": true,
      "editor.defaultFormatter": "ms-python.autopep8"
    },
`

Comment (Maurier) [reply api/models/schema.py]: I have undone the indentations now.

**Pull request merged**